ppl/zqd/handlers_test.go

```diff
@@ -758,7 +761,7 @@ func TestIndexSearch(t *testing.T) {
 		},
 	})
 	require.NoError(t, err)
-	_, err = conn.LogPost(context.Background(), sp.ID, nil, babble)
+	_, err = conn.LogPost(context.Background(), sp.ID, nil, babbleSorted)
```

Comment:
Maybe add a comment explaining why babbleSorted is used here?

Reply:
Sure